test/RootCloseWrapperSpec.js

```diff
@@ -100,6 +106,53 @@ describe('RootCloseWrapper', function () {
 
       expect(outerSpy).to.have.not.been.called;
       expect(innerSpy).to.have.been.calledOnce;
+
+      assert.oneOf(innerSpy.getCall(0).args[0].type, ['click', 'mousedown']  );
+    });
+  }
+
+  function shouldCloseOnKeyUp(eventName) {
```

Comment:
this shouldn't be a function, since you only call it once. Instead inline the tests below in the keydown describe block